libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -112,6 +121,29 @@ bool AP_OAPathPlanner::pre_arm_check(char *failure_msg, uint8_t failure_msg_len)
     return true;
 }
 
+bool AP_OAPathPlanner::start_thread()
```

Comment:
the start_thread() function seems to have some duplicate code that's already in mission_avoidance.  I think we should remove one of them (probably the one in mission_avoidance).

Reply:
ahh yeah, I missed that when rebasing when you added that handy OA_ERROR result thing. It's also unreachable now too. Anyways, removed.